project/src/components/offer-list/offer-list.tsx

```diff
@@ -0,0 +1,26 @@
+import {useState} from 'react';
+import {Offers} from '../../types/offer';
+import Card from '../card/card';
+
+type OfferListProps = {
+  offers: Offers;
+  className: string;
+}
+
+export default function OfferList({offers, className}: OfferListProps): JSX.Element {
+
+  const [activeCard, setActiveCard] = useState<number | null>(null);
+
+  return (
+
+    <div className={`${className}`} data-active-card={activeCard}>
+      {offers && offers.map((offer) => (
```

Comment:
Почему offers может не быть? Тогда тип должен стоять корректный, но по факту они всегда будут, если будет ошибка до этого компонента рендер не дойдет так как мы обработаем это раньше
```suggestion
      {offers.map((offer) => (
```